cmd/zetae2e/get_zetaclient_bootstrap.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/spf13/cobra"
+	"github.com/zeta-chain/node/pkg/rpc"
+	"github.com/zeta-chain/node/pkg/sdkconfig"
+	observertypes "github.com/zeta-chain/node/x/observer/types"
+	"gitlab.com/thorchain/tss/go-tss/conversion"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func NewGetZetaclientBootstrap() *cobra.Command {
+	var ConfigureZetaclientBootstrapCmd = &cobra.Command{
+		Use:   "get-zetaclient-bootstrap",
+		Short: "get bootstrap address book entries for zetaclient",
+		RunE:  getZetaclientBootstrap,
+	}
+
+	return ConfigureZetaclientBootstrapCmd
+}
+
+func getZetaclientBootstrap(cmd *cobra.Command, _ []string) error {
+	sdkconfig.SetDefault(true)
+	rpcClient, err := rpc.NewGRPCClients(
+		"zetacore0:9090",
```

Comment:
This is a value in the config, why not reading it from here. We can also put this command under `local` to make explicit it's only used for local testing

Reply:
> This is a value in the config, why not reading it from here.

The e2e config isn't in currently available in the zetanode/zetaclient container.

> We can also put this command under local to make explicit it's only used for local testing

Ok

Reply:
If it's in local then it's fine